test/scripts/test-wrapper.go

```diff
@@ -14,7 +14,6 @@ import (
 
 type groupName string
 type addonName string
-type groups map[groupName][]addonName
```

Comment:
I'm curious what the reason for removing this type alias was, it seems somewhat arbitrary? :thinking:

Reply:
It is somehow arbitrary tbh. I just saw that the type was only being used once, anyway, and so I didn't see a reason for it to exist.

Reply:
I think the idea originally was just about context and expression, but no worries changing it is not going to cause any problems.